src/screens/LoginScreen/index.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { FC } from 'react';
+import { View } from 'react-native';
+import { style } from './style';
+import { SignInButton } from '../../components/SignInBotton';
+
+export const LoginScreen: FC = () => {
```

Comment:
all logic related signIn and dispatch actions we have to pass inside LoginScreen. button will get props

Reply:
yes it's my mistake